R/options.r

```diff
@@ -12,6 +12,7 @@ getenv_default <- function(varname, default) {
 #' @export
 jupyter_option_defaults <- list(
     jupyter.log_level = as.integer(getenv_default('JUPYTER_LOG_LEVEL', 1L)),
+    jupyter.logfile = getenv_default('JUPYTER_LOGFILE'),
```

Comment:
won’t this say “missing argument, no default”?

maybe we should change the definition of `getenv_default` to `function(varname, default = NULL)`?